src/DocumentTemplate/DT_In.py

```diff
@@ -837,7 +837,7 @@ def sort_sequence(self, sequence, md):
             by = SortBy(multsort, sf_list)
             s.sort(by)
         else:
-            s.sort()
+            s.sort(key=lambda k: k[0])
```

Comment:
Can you add a comment here that the explicit key is needed for the case where more than one tuple in the list has the same sort key, to prevent attempting to compare the second item (maybe note Py3k)?

Reply:
Thanks for formulating and explaining the issue. I did not know that sort moves to the second item when the first is the same.